examples/oft-aptos/aptos.layerzero.config.ts

```diff
@@ -0,0 +1,161 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions-v3'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities-v3'
+import { bscConfig } from './evmOAppConfig'
+import { ethConfig } from './evmOAppConfig'
+
+import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+
+const ethereumContract: OmniPointHardhat = {
+    eid: EndpointId.ETHEREUM_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const bscContract: OmniPointHardhat = {
+    eid: EndpointId.BSC_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const aptosContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_V2_TESTNET,
+    contractName: 'oft',
+}
+
+const ethereumSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.ETHEREUM_V2_SANDBOX,
+    contractName: 'MyOFT',
+}
+
+const bscSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.BSC_V2_SANDBOX,
+    contractName: 'MyOFT',
+}
+
+const aptosSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_V2_SANDBOX,
+    contractName: 'oft',
+}
+
+const config: OAppOmniGraphHardhat = {
+    contracts: [
+        {
```

Comment:
did we confirm that Sepolia is wired to Aptos Testnet? If not can we leave here only Aptos Testnet and BSC Testnet?

Reply:
Sepolia is not wired. I just had it there for testing with `SimpleMessageLib` which always returns True on endpoint id validation